docker-images/zrok-private-access/docker-compose.yml

```diff
@@ -0,0 +1,64 @@
+version: '3'
+services:
+  zrok-enable-init:
+    image: busybox
+    # matches uid:gid of "nobody" in zrok container image
+    command: chown -Rc 65534:65534 /mnt/.zrok
+    user: root
+    volumes:
+      - zrok_env:/mnt/.zrok
+  zrok-enable:
+    image: docker.io/openziti/zrok:enable-headless  # FIXME: resume :latest if >= :0.3.2
+    depends_on:
+      zrok-enable-init:
+        condition: service_completed_successfully
+    entrypoint:
+      - bash
+      - -c
+      - |
+        if [[ -s /mnt/.zrok/environment.json ]]; then
+          echo "INFO: noop: zrok environment is already enabled"
+          exit 0
+        else
+          echo "INFO: running: zrok $$(sed -E 's/\S{12,}/************/' <<< $${@})"
```

Comment:
isn't this fragile? should it just be doing: `sed -E "s/$ZROK_ENABLE_TOKEN/***"` instead?

Reply:
I would like the regex to be more strict, but I'm not certain the character class used to generate random tokens is safe for a pattern match. If it has any characters with special meaning for regex then it won't match and will be logged as plaintext.

Reply:
NM. zrok is just using Nanoid with alnum character class. Your idea will work.

https://github.com/openziti/zrok/blob/main/cmd/zrok/adminGenerate.go#L76